tests/integration/cmd_link_unlink_test.go

```diff
@@ -38,6 +54,104 @@ var _ = Describe("odo link and unlink command tests", func() {
 		})
 	})
 
+	Context("When running an etcd operator backed service", func() {
```

Comment:
I think these tests should be in https://github.com/openshift/odo/blob/main/tests/integration/operatorhub/cmd_service_test.go file. A similar linking scenario is already present in that file.